targets/ath79-generic

```diff
@@ -266,6 +266,10 @@ device('tp-link-archer-c6-v2', 'tplink_archer-c6-v2', {
 	packages = ATH10K_PACKAGES_QCA9888,
 })
 
+device('tp-link-archer-c7-v2', 'tplink_archer-c7-v2', {
+	packages = ATH10K_PACKAGES_QCA98880
```

Comment:
There seems to be a typo here: `QCA98880`

Reply:
You're right, i was a little to fast there. Fixed.